skylark/skylark.bzl

```diff
@@ -174,7 +88,7 @@ _skylark_doc_attrs = {
 
 skylark_doc = rule(
     _skylark_doc_impl,
-    attrs = dict(_skylark_common_attrs.items() + _skylark_doc_attrs.items()),
+    attrs = _skylark_doc_attrs,
```

Comment:
Does this really need to be in a separate dictionary?  Why not just inline it directly here?